include/cuda/std/detail/libcxx/include/optional

```diff
@@ -351,9 +412,9 @@ struct __optional_storage_base : __optional_destruct_base<_Tp>
     }
 };
 
-// optional<T&> is currently required ill-formed, however it may to be in the
-// future. For this reason it has already been implemented to ensure we can
-// make the change in an ABI compatible manner.
+// optional<T&> is currently required to be ill-formed. However, it may
```

Comment:
Right now I am in favor of just remmoving that one. We are not really caring about ABI and I do not really see why it would break ABI because you cannot use an optional with a reference type today